test/builtin_dataset_mocks.py

```diff
@@ -95,6 +95,8 @@ def parametrize_dataset_mocks(*dataset_mocks, marks=None):
     for mock in dataset_mocks:
         if isinstance(mock, DatasetMock):
             mocks[mock.name] = mock
+        elif isinstance(mock, collections.abc.Sequence):
```

Comment:
@NicolasHug we already agreed that the way we are parametrizing needs to change. If you don't oppose I wouldn't do that in this PR though.